dash_table_base/__init__.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import print_function as _
```

Comment:
What follows is problematic for a dev environment where the developer uses `pip install -e ./dash-table` to load the table package into his venv.

So... the dash generator is having some trouble here. With the generated *.py files missing, the package can no longer load correctly. This was apparently already an issue when this was done for the renderer, but since **all** the files are being generated in that case, the empty folder is handled correctly when Python does the `install_requires` check for Dash.

These files are not generated, moving them to a baseline folder that gets copied into `dash_table` at the right moment. That's the best option I've found so far apart from making the generator independent from Dash to resolve the dependencies chicken & egg problem.

The generation itself had to be modified also (package.json) because the R generation code depends on the baseline artifacts being present, which can't be, because otherwise the generator does not run at all..

So we generate Python, copy baseline, generate again with R. Not efficient but it works, it's two steps forward, one step backward.